src/include/daos/object.h

```diff
@@ -402,6 +402,8 @@ enum daos_io_flags {
 	DIOF_EC_RECOV		= 0x8,
 	/* The key existence. */
 	DIOF_CHECK_EXISTENCE	= 0x10,
+	/* The RPC will be sent to specified redundancy group. */
+	DIOF_TO_SPEC_GROUP	= 0x20,
```

Comment:
DIOF_TO_SPEC_GROUP the name seems a little confuse, as in the rebuild case seems also for a specific shard. may confirm with @wangdi1

Reply:
It is wangdi suggested that rebuild will fetch from specified group, not a specified shard.